Makefile

```diff
@@ -192,6 +192,35 @@ prepare-release-candidate:
 print-version:
 	@go run pkg/version/generate/release_generate.go print-version
 
+.PHONY: upload-github
+upload-github:
+	@echo "Releasing version $(eksctl_version) in $(git_org)/$(git_repo)"
+	@echo "Check draft exists..." && github-release info --user $(git_org) --repo $(git_repo) --tag $(eksctl_version)
+	github-release upload --user $(git_org) --repo $(git_repo) --tag $(eksctl_version) --file dist/eksctl_Windows_amd64.zip --name eksctl_Windows_amd64.zip
+	github-release upload --user $(git_org) --repo $(git_repo) --tag $(eksctl_version) --file dist/eksctl_Darwin_amd64.tar.gz --name eksctl_Darwin_amd64.tar.gz
+	github-release upload --user $(git_org) --repo $(git_repo) --tag $(eksctl_version) --file dist/eksctl_Linux_amd64.tar.gz --name eksctl_Linux_amd64.tar.gz
+
+.PHONY: publish-github
+publish-github: upload-github
+	github-release publish --user $(git_org) --repo $(git_repo) --tag $(eksctl_version)
+
+.PHONY: publish-rc-github
+publish-rc-github: upload-github
+	github-release release --user $(git_org) --repo $(git_repo) --tag $(shell eksctl version) --pre-release
+
+.PHONY: publish-homebrew
+publish-homebrew:
```

Comment:
Should we call this as part of `publish-github`? (In that case we could maybe rename the targets `publish-release` and `publish-rc`? Or add these which would call `*-github` and `publish-homebrew`?)

Reply:
I thought about it. I prefer to leave it de-coupled for now until we start using it. Then we can decide how to use it locally or from CircleCI. We can always refactor these later when we know more about how we want to use them.

Reply:
Sounds good to me! 👍